CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3.0 - 2023-01-06
```

Comment:
Please change this to "UNRELEASED". We will add the version number when we publish in case other code changes are added prior to release.